ProcessMaker/processmaker
# Bugfix/four 8091

## Ticket 
- [FOUR-8091](https://processmaker.atlassian.net/browse/FOUR-8091)

## Issue & Reproduction Steps
We don't have translated labels in the Designer -> Templates tab or any of the Ellipsis menu options.

## Solution
- Add translation in the menu options for the Templates tab.
- Add translations in the Ellipsis menu on the rest of the tabs.

## How to Test
- Install the translations package
- Log in with another language. Eg: Spanish
- Click on Designer 
- In the Process click on the Ellipsis menu and make sure all the options are translated
- Click on the Templates  tab 
- Click on the ellipsis menu

## Current Behavior: 
- The menu is not translated.

## Expected Behavior:
- The menu should be translated into Spanish, French, and German depending on the User's language.
![Screenshot 2023-05-03 at 10 53 13 AM](https://user-images.githubusercontent.com/5769433/236003190-dedacce5-6bb5-4674-9b28-b22e38247b10.png)
![Screenshot 2023-05-03 at 10 51 51 AM](https://user-images.githubusercontent.com/5769433/236003193-63907f2d-f4d5-45cf-8907-b53806aa8310.png)
![Screenshot 2023-05-03 at 10 51 41 AM](https://user-images.githubusercontent.com/5769433/236003196-991964bc-f7be-4894-86e1-88596510aa96.png)


## Code Review Checklist


[FOUR-8091]: https://processmaker.atlassian.net/browse/FOUR-8091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
